CHANGELOG.md

```diff
@@ -0,0 +1,46 @@
+# Changelog
+
+## [Unreleased]
```

Comment:
```suggestion
```

Think we can remove this heading

Reply:
I added some placeholder text to this heading to make sure it's not an empty heading without content. Just want to make sure we have a designated place to park changelogs for a future release.